sled-agent/src/bootstrap/params.rs

```diff
@@ -19,6 +43,15 @@ pub struct SledAgentRequest {
     /// Uuid of the rack to which this sled agent belongs.
     pub rack_id: Uuid,
 
+    /// Information about internet gateway to use
+    // NOTE: This information is currently being configured and sent from RSS,
+    // but it contains dynamic information that could plausibly change during
+    // the duration of the sled's lifetime.
+    //
+    // Longer-term, it probably makes sense to store this in CRDB and transfer
+    // it to Sled Agent as part of the request to launch Nexus.
```

Comment:
To elaborate a bit on this comment, I believe this PR accomplishes https://github.com/oxidecomputer/omicron/issues/1527 by moving the gateway information from:

"must be known at package time" (bundled with Sled Agent)
to
"must be known at RSS time" (set once when configuring Sled Agent)

It probably makes sense to migrate this further, to:
"must be known when launching Nexus (and can be modified on a live system)"